misc/graylog.conf

```diff
@@ -482,6 +482,9 @@ inputbuffer_ring_size = 65536
 inputbuffer_processors = 2
 inputbuffer_wait_strategy = blocking
 
+# Manually stopped inputs are not auto-restarted. To re-enable the previous behavior, set autostart_inputs to true.
+#autostart_inputs = true
```

Comment:
I think the name might be misleading. It sounds as if this config setting would influence the general autostart behaviour (for new inputs), not just on server restart. Maybe we should use `auto_restart_inputs` instead?

Reply:
Fixed - forgot to include graylog.conf in the previous commit :/